src/tools/kata-ctl/src/arch/x86_64/mod.rs

```diff
@@ -43,9 +58,8 @@ mod arch_specific {
         Ok(())
     }
 
-    pub fn check() -> Result<()> {
-        println!("INFO: check: x86_64");
-
+    pub fn check(args: &str) -> Result<()> {
+   
```

Comment:
Nit: There is extraneous whitespace on this line (although it doesn't show on GH). I'd recommend getting `rustfmt` set up to automatically remove things like that.

Reply:
Yeah I noticed cargo fmt fixed this one